llumnix/backends/migration_backend_interface.py

```diff
@@ -39,3 +41,24 @@ def do_send(self, dst_handle, blocks: List[int]):
     @abstractmethod
     def do_recv(self, src_handle, blocks: List[int]):
         raise NotImplementedError
+
+class CacheMigrationBackend(MigrationBackendBase):
+    def __init__(self, num_cache, cache_shape, cache_dtype, cache_device, pin_memory, *args, **kwargs):
```

Comment:
I want to know that, if it means that if num_cache == 2, the maximum number of concurrent migrations of one instance is 2, and if num_cache == 3, the maximum number is 3, right?

Reply:
yes